cmd/crank/version/version.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2023 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package version contains version cmd
+package version
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	"github.com/alecthomas/kong"
+	"github.com/pkg/errors"
+
+	"github.com/crossplane/crossplane/internal/version"
+)
+
+const (
+	errGetCrossplaneVersion = "unable to get crossplane version"
+)
+
+// Cmd represents the version command.
+type Cmd struct {
+	Client bool `env:"" help:"If true, shows client version only (no server required)." json:"client,omitempty"`
+}
+
+// BeforeApply sets default values and parses flags.
+func (c *Cmd) BeforeApply() error {
+	flag.BoolVar(&c.Client, "client", false, "If true, shows client version only (no server required).")
+	flag.Parse()
```

Comment:
This looks a little suspicious. Isn't it possible to define a flag using kong, without needing to use the flag library?

Reply:
removed